pfcpiface/bess.go

```diff
@@ -244,6 +257,40 @@ func (b *bess) summaryLatencyJitter(uc *upfCollector, ch chan<- prometheus.Metri
 
 }
 
+func (b *bess) readReportNotification() {
+	log.Println("report bess notification start")
+	for {
+		select {
+		case d := <-b.reportNotification:
+			fseid := binary.LittleEndian.Uint64(d)
+			handleDigestReport(fseid, b.pfcpConn, b.udpConn, b.udpAddr)
+
+		default:
+			time.Sleep(2 * time.Second)
+		}
+	}
+}
```

Comment:
As said above this code is preferable if placed in pfcp layer instead of fastpath interface implementation